Dev to master 24.03 (#80)
ElrondNetwork/docs.elrond.com: docs/validators/delegation-manager.md

}
```

An example for the `Data` field that sets the name to `"Elrond Staking"`, the website to `"elrond.staking"` and the keybase.io identifier to `"elrondstaking"` is: 
```
    "setMetaData" + 
    "@456c726f6e64205374616b696e67"     //Elrond Staking
    "@656c726f6e642e7374616b696e67"     //elrond.staking     
    "@656c726f6e647374616b696e67"       //elrondstaking
```

:::important

> Setting the keybase.io identity of the staking pool in the metadata is the **first step** in connecting the delegation contract and a keybase.io identity. The second step is explained in the next chapter [display information](/validators/delegation-manager#display-information) where the inverse connection is made: from the keybase.io identity to the delegation contract address. 

Reviewer: ```suggestion
Setting the keybase.io identity of the staking pool in the metadata is the **first step** in connecting the delegation contract and a keybase.io identity. The second step is explained in the next section, [Display information](/validators/delegation-manager#display-information), where the inverse connection is made: from the keybase.io identity to the delegation contract address. 
```

Author: fixed

:::

> ### Display Information

Reviewer: ```suggestion
### Display information
```

Author: fixed

To customize the information for your delegation contract that will be available in the lists displayed on the delegation pages both in Maiar and the web wallet some aditional information has to be added on the keybase.io account. Please fill in the **avatar picture** and edit the profile providing the **name** and **bio**. This information together with the **service fee, percentage filled** and **APR** will be displayed for every delegation contract on the delegation pages in the web wallet and Maiar. If these information cannot be found a generic logo and the delegation contract's address is displayed.

> In order to complete the matching between the delegation contract and keybase.io identity of the staking pool an empty file with the name set to the delegation contract's address has to be added in the `/public/<keybase.io identity>/elrond/` folder. An example for the path to the empty file for the `"elrondstaking"` keybase.io identity would be `public/elrondstaking/elrond/erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py`.

Reviewer: It might be better to put the path to the file as a code block using three backticks so it won't be wrapped when displayed by the browser.

Author: fixed

:::important
The **second step** in connecting the delegation contract and the keybase.io identity is finalized by adding the empty file that has the name equal to the delegation contract's address and certifying that the referenced keybase.io identity set in the [metadata](/validators/delegation-manager#metadata) also references the delegation contract address. This way the connection from both directions is sealed.
:::

:::tip

> To be able to connect a **testnet** or **devnet** contract to a keybase.io identity a new folder named `"testnet"` for the testnet or `"devnet"` for the devnet has to be created inside the `/elrond` folder. An example for the same delegation contract on the `testnet` would be `public/elrondstaking/elrond/testnet/erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py`.

Reviewer: It might be better to put the path to the file as a code block using three backticks so it won't be wrapped when displayed by the browser.

Author: fixed

:::

An example of how the delegation contract will be displayed based on the information provided in the keybase.io is provided below.

![stakingpool](/img/stakingpool.png)

:::important

> To be able to connect also the validators to a specific keybase.io staking pool identity, two steps have to be completed as well:

Reviewer: ```suggestion
To also connect the validators themselves to a specific keybase.io staking pool identity, two additional steps have to be completed:
```

1. Create an empty file with the name set to the `"<BLS key>"` for every validator and add the empty file to the `/elrond` folder on your keybase.io identity: `public/<keybase.io identity>/elrond/<BLS key>`

> 2. Set the `Identity` of the validator in the `config/prefs.toml` file to the keybase.io staking pool identity.

Reviewer: ```suggestion
2. Set the `Identity` of each validator in the `config/prefs.toml` file to the keybase.io staking pool identity.
```
